packages/babel-helper-replace-supers/src/index.js

```diff
@@ -80,6 +80,10 @@ const unshadowSuperBindingVisitor = traverse.visitors.merge([
   environmentVisitor,
   {
     Scopable(path, { refName }) {
+      if (path.type === "ClassDeclaration" || path.type === "ClassExpression") {
+        path.skip();
+        return;
+      }
```

Comment:
This will probably break things like
```js
class Foo extends Bar {
  constructor() {
    super();
    class X {
      [(() => {
        let Foo;
        super.method();
      })()]() {}
    }
}
```
which should call `Bar.prototype.method()`

Reply:
👌 Got it